docs/tutorials/text-classification-tutorial.mdx

```diff
@@ -0,0 +1,295 @@
+```
+
+```
+Query OK, 24784 rows affected, 17300 warnings (0.26 sec)
+Records: 26952  Deleted: 0  Skipped: 2168  Warnings: 17300
+```
+
+---
+
+- Go ahead and confirm your import:
+
+`SELECT * FROM tweets LIMIT `0`
+
+```sql
+------------------------------------------------------------+
+    -> | id | hate_speech | offensive | neither | tweet                                                                                                                                         |
+    -> +----+-------------+-----------+---------+-----------------------------------------------------------------------------------------------------------------------------------------------+
+    -> |  0 |           0 |         0 |       3 | !!! RT @mayasolovely: As a woman you shouldn't complain about cleaning up your hou |. &amp; as a man you should always take the trash out...
+    '> |  1 |           0 |         3 |       0 | !!!!! RT @mleew17: boy dats cold...tyga dwn bad for cuffin dat hoe in the 1st plac                                                        |
+    '> |  2 |           0 |         3 |       0 | !!!!!!! RT @UrKindOfBrand Dawg!!!! RT @80sbaby4life: You ever fuck a bitch and she                     |confused as shit
+    '> +----+-------------+-----------+---------+-----------------------------------------------------------------------------------------------------------------------------------------------+
+```
+---
+
+## Connecting To MySQL Datsource
+
+- Now that your MySQL database and table are configured, use *ngrok* to expose your local MySQL server:
+
+`ngrok tcp 3306` 
+
+- Next, head to `https://cloud.mindsdb.com/editor`.
+
+Using SQL's, `CREATE DATABASE` statement and the credentials supplied by `ngrok` output, securely connect to MySQL right inside the MindsDB Cloud Editor:
+
+```sql
+CREATE DATABASE twitter_data  --- display name for database.
+WITH ENGINE = 'mysql',        --- name of the mindsdb handler
+PARAMETERS = {
+  "user": "root",             --- your mysql username
+  "password": "MyN3wP4ssw0rd",    --- your mysql password  
+   "host": "6.tcp.ngrok.io",    ---ngrok host credentials 
+  "port": "10105",                  ---ngrok forwarded port
+  "database": "nlp_mindsdb"          --- database name
+};
+```
+
+You should receive the following output,
+`Query Successfully Executed`
+
+- Go ahead and confirm the new database includes the tweets table you built in your command line:
+
+```sql
+SELECT *
+FROM twitter_data.tweets
+LIMIT 10;
+```
+
+***your output will look like this, but with 10 rows***
+
+| id  | hate_speech | offensive | neither | tweet |
+| --- | ----------- | --------- | ------- | ----- |
+| 0 | 0 | 0 | 3 | !!! RT @mayasolovely: As a woman you shouldn't complain about cleaning up your house. &amp; as a man you should always take the trash out... |
+| 1 | 0 | 3 | 0 | !!!!! RT @mleew17: boy dats cold...tyga dwn bad for cuffin dat hoe in the 1st place!! |
```

Comment:
Please remove all offensive/vulgar words and examples from this tutorial. MindsDB documentation shouldn't contain any negative language or hate speech.